### PR TITLE
XW-1205 | Move Nielsen initial PV from Ember

### PR DESCRIPTION
**Pull request by vforge**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1205

## Description

Move initial Nielsen PV from Ember

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: server/app/views/_partials/tracking-nielsen.hbs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<script>
+	(function () {
+		var sendNielsenPageView = function () {
+			if (!window.NOLCMB || !window.NOLCMB.getInstance) {
+				return;
+			}
+
+			var nielsen = M.prop('tracking.nielsen'),
```
Comment (bognix) [inline server/app/views/_partials/tracking-nielsen.hbs]: move var to top

**Comment by bognix** [comment]
LGTM

File: server/app/views/_partials/tracking-nielsen.hbs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			if (!M.prop('queryParams.noexternals')) {
+				nielsen = M.prop('tracking.nielsen');
+				scriptTag = document.createElement('script');
+				firstScriptInDocument = document.getElementsByTagName('script')[0];
+
+				if (nielsen.enabled) {
+					delete window.NOLCMB;
+
+					scriptTag.async = 1;
+					scriptTag.src = 'http://secure-dcr-cert.imrworldwide.com/novms/js/2/ggcmb500.js';
```
Comment (hakubo) [inline server/app/views/_partials/tracking-nielsen.hbs]: it looks like they have a nice api - do we still need to load their script on every page view?

**Comment by vforge** [comment]
Ping @drets @nandy-andy for additional LGTM

File: front/common/utils/track.js [hunk 3]
```diff
@@ -201,7 +199,11 @@ export function trackPageView(adsContext) {
 		M.prop('initialPageView', false);
 	} else {
 		window.trackQuantservePageView();
+<<<<<<< HEAD
```
Comment (rogatty) [inline front/common/utils/track.js]: Uhmmm...

Comment (nandy-andy) [inline front/common/utils/track.js]: :see_no_evil:

File: server/app/views/_partials/tracking-nielsen.hbs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+			if (!M.prop('queryParams.noexternals')) {
+				nielsen = M.prop('tracking.nielsen');
+				scriptTag = document.createElement('script');
+				firstScriptInDocument = document.getElementsByTagName('script')[0];
+
+				if (nielsen.enabled) {
+					delete window.NOLCMB;
+
+					scriptTag.async = 1;
+					scriptTag.src = 'http://secure-dcr-cert.imrworldwide.com/novms/js/2/ggcmb500.js?' + Date.now();
```
Comment (drets) [inline server/app/views/_partials/tracking-nielsen.hbs]: :+1:

File: front/common/utils/track.js (hunk 3 above)
Comment (vforge) [inline front/common/utils/track.js]: Ops!

Comment (rogatty) [inline front/common/utils/track.js]: Yeah, I thought it was them!

**Comment by vforge** [comment]
tested, works

**Pull request merged**